src/bounded/bdd.rs

```diff
@@ -596,10 +599,10 @@ assert always (forall N1:node, N2:node. holds_lock(N1) & holds_lock(N2) -> N1 =
         let mut module = crate::fly::parse(source).unwrap();
         let universe = HashMap::from([("node".to_string(), 2)]);
 
-        let bug = check(&mut module, &universe, 12)?;
+        let bug = check(&mut module, &universe, Some(12))?;
         assert_eq!(CheckerAnswer::Counterexample, bug);
```

Comment:
You could copy this test with the same assertion for `depth = None` (I wouldn't even bother with a let binding, just add an `assert_eq!`).